src/components/Home/Home.tsx

```diff
@@ -1,13 +1,27 @@
-import { Title } from "@deskpro/app-sdk";
+import { Fragment } from "react";
+import { map, isEmpty } from "lodash-es";
+import { HorizontalDivider } from "@deskpro/app-sdk";
+import { Container, NotFound } from "../common";
+import { DeviceItem } from "../DeviceItem";
 import type { FC } from "react";
+import type { DeviceType } from "../../types";
 
-type Props = {
-  //..
+export type Props = {
+  devices: DeviceType[];
 };
 
-const Home: FC<Props> = () => {
+const Home: FC<Props> = ({ devices }) => {
   return (
-    <Title title="Home Page"/>
+    <Container>
+      {isEmpty(devices) ? (
+        <NotFound text="No devices found" />
+      ) : map(devices, (device) => (
```

Comment:
Replace `map` with JS/TS native `map` - no need for lodash.

Reply:
done